rent-a-car/src/main/java/xws/tim16/rentacar/service/CustomUserDetailsService.java

```diff
@@ -189,19 +204,15 @@ private User getUserById(Long userId) {
     }
 
     private Collection<? extends GrantedAuthority> getAuthorities(
-            Collection<Role> roles) {
+            User user) {
 
-        return getGrantedAuthorities(getPrivileges(roles));
+        return getGrantedAuthorities(getPrivileges(user.getPrivileges()));

```

Comment:
Authority is actually role, so this shouldn't have been changed

Reply:
But this method has already returning privileges,i just changed input parameter because i changed model